module_prototyper/models/module_prototyper_api_version.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Onestein (<http://www.onestein.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, fields
+
+
+class ModulePrototyperApiVersion(models.Model):
+    _name = 'module_prototyper.api_version'
+
+    name = fields.Char(
+        string='Name'
```

Comment:
`If the technical name of the field (the variable name) is the same to the string of the label, don't put string parameter for new API fields, because it's automatically taken. If your variable name contains "_" in the name, they are converted to spaces when creating the automatic string and each word is capitalized. (example: old api 'name': fields.char('Name', ...) new api 'name': fields.Char(...))`
https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#fields